exonum_client/protoc.py

```diff
@@ -109,6 +129,7 @@ def compile(self, path_in: str, path_out: str, include: str = None) -> None:
         else:
             logger.error("Error acquired while compiling files: %s. Files: %s.", stderr.decode("utf-8"), proto_files)
 
-        modules = [proto_path.replace(".proto", "") for proto_path in proto_files]
-        for file in filter(lambda f: f.endswith(".py"), os.listdir(path_out)):
-            self._modify_file("{}/{}".format(path_out, file), modules)
+        modules = [proto_path.split("/")[-1].replace(".proto", "") for proto_path in proto_files]
+        for file in glob.glob(path_out + "/**/*.py", recursive=True):
```

Comment:
Nit: Maybe rename `_find_proto_files` to `_find_files_recursively` and add a file extension to the parameters, so it can be reused like `_find_files_recursively(path, ".proto")` and `_find_files_recursively(path, ".py")`?